views/case-edit.html

```diff
@@ -62,9 +62,9 @@ <h2 class="fieldset-header">{{t "process_sectionlabel"}}</h2>
     {{> article-edit-select name="facilitator_training" }}
     {{> article-edit-select name="facetoface_online_or_both" }}
     {{> edit-multi-select ranked=true name="participants_interactions" max=3 }}
-    {{> edit-multi-select name="learning_resources" }}
-    {{> edit-multi-select name="decision_methods" }}
-    {{> edit-multi-select name="if_voting" }}
+    {{> edit-multi-select name="learning_resources" max=5 }}
```

Comment:
rather than setting a max inline here, let's set the default max to 5 in `edit-multi-select.js`.

something like this in https://github.com/participedia/api/blob/homepage-2020/public/js/edit-multi-select.js#L113 : 
```
const DEFAULT_MAX_ITEMS = 5;
const maxItems = currentList.getAttribute("data-max") || DEFAULT_MAX_ITEMS;


```